test/test.ts

```diff
@@ -75,8 +78,20 @@ test('udp-device-connects', (t) => {
     scanConfig.broadcastAddress
   );
 
+  loadHomeApp('../home-app/bundle');
```

Comment:
what about including the test app implemention inline rather than loading it from the filesystem?

Reply:
I will make this change in #30